2025-02-04 23:20:06,924 - INFO - Namespace(device='cuda:0', channel_weights=[1.0, 2.0, 1.0, 1.0], batch_size=4, num_classes=4, pretrained=False, pretrained_weights='./Synapse_weights_1u1d', lr=0.0001, epochs=500, warmup_epochs=10, dataset_mode='MnMs', max_channel=256, l2_norm=1e-07, patience=15, wandb=True, cache=True, ed_es_only='', log_dir='../../de_logistics/MnMs_1UNetlinear_02-04-23-20-05/', log_file_path='../../de_logistics/MnMs_1UNetlinear_02-04-23-20-05/training.log')
Epoch 1/500 [Training]:  22%|████████▊                                | 61/282 [00:20<01:15,  2.91batch/s]
Traceback (most recent call last):
  File "/data2/lyw/deunet_mem_based/train.py", line 226, in <module>
    main(args, net)
  File "/data2/lyw/deunet_mem_based/train.py", line 68, in main
    train_acc, train_losses = train_epoch(args, net, optimizer, train_loader, device, epoch, scheduler)
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data2/lyw/deunet_mem_based/train.py", line 139, in train_epoch
    train_losses, train_acc, _= process_epoch(args, net, train_loader, device, 'train', optimizer, scaler, scheduler)
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data2/lyw/deunet_mem_based/train.py", line 107, in process_epoch
    evaluator = Evaluator(args, net, pred, label, label_one_hot, valid_mask, weights=weights, train_eval=True)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data2/lyw/deunet_mem_based/utils/metrics.py", line 149, in __init__
    self.confuse_matrix = self.compute_batch_confuse_matrix()
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data2/lyw/deunet_mem_based/utils/metrics.py", line 169, in compute_batch_confuse_matrix
    pred_image = np.argmax(pred[i], axis=0)  # (height, width)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lyw/anaconda3/envs/unet/lib/python3.11/site-packages/numpy/_core/fromnumeric.py", line 1298, in argmax
    return _wrapfunc(a, 'argmax', axis=axis, out=out, **kwds)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lyw/anaconda3/envs/unet/lib/python3.11/site-packages/numpy/_core/fromnumeric.py", line 57, in _wrapfunc
    return bound(*args, **kwds)
           ^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
